queries/usd/highlights.scm

```diff
@@ -0,0 +1,228 @@
+(None) @constant.builtin
+(asset_path) @text.uri
+(bool) @boolean
+(comment) @comment @spell
+(custom) @function.builtin
+(float) @float
+(integer) @number
+(namespace_identifier) @namespace
+(orderer) @function.call
+(prim_path) @string.special
+(uniform) @function.builtin
+
+[
+  "class"
+  "def"
+  "over"
+] @keyword.function
+
+["(" ")" "[" "]" "{" "}"] @punctuation.bracket
+[":" ";" "."] @punctuation.delimiter
+
+[
+  "="
+] @operator
+
+;; Simple variables
+(attribute_assignment
+ (identifier) @variable
+)
+(attribute_declaration
+ (identifier) @variable
+)
+(relationship_declaration
+ (identifier) @variable
+)
+(relationship_assignment
+ (identifier) @variable
+)
+(qualified_identifier
+  (identifier) @variable)
```

Comment:
This whole section should just be `(identifier) @variable`

Reply:
This is done now, thank you